app/src/main/java/ch/epfl/sweng/swenggolf/offer/ListOfferActivity.java

```diff
@@ -31,24 +31,26 @@
  */
 public class ListOfferActivity extends FragmentConverter {
 
-    public static final List<Offer> offerList = new ArrayList<>();
+    // The list is public, static and final so that it can be used in tests.
+    public static final List<Offer> OFFER_LIST = new ArrayList<>();
```

Comment:
You explained it to us, but I still think that a mutable list that is final is not a very good idea (i.e. it doesn't make it immutable). Are you sure you cannot do without in your tests (e.g. put a public getter that returns an immutable view of the list)?